x-pack/plugins/observability_solution/observability_onboarding/public/application/onboarding_flow_form/use_virtual_search_results.ts

```diff
@@ -6,14 +6,47 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { useHistory } from 'react-router-dom';
+import { reactRouterNavigate, useKibana } from '@kbn/kibana-react-plugin/public';
 import { CustomCard } from '../packages_list/types';
+import { ObservabilityOnboardingAppServices } from '../..';
 
 export function useVirtualSearchResults(): CustomCard[] {
   const {
-    services: { application },
-  } = useKibana();
+    services: {
+      application,
+      context: { isServerless },
+    },
+  } = useKibana<ObservabilityOnboardingAppServices>();
+  const history = useHistory();
+  const { href: firehoseUrl } = reactRouterNavigate(history, `/firehose/${location.search}`);
   const getUrlForApp = application?.getUrlForApp;
+  const firehoseQuickstartCard: CustomCard = {
+    id: 'firehose-quick-start',
+    type: 'virtual',
+    title: i18n.translate('xpack.observability_onboarding.packageList.uploadFileTitle', {
+      defaultMessage: 'AWS Firehose',
+    }),
+    release: 'preview',
+    description: i18n.translate(
+      'xpack.observability_onboarding.packageList.uploadFileDescription',
+      {
+        defaultMessage: 'Collect logs and metrics from Amazon Web Services (AWS).',
+      }
+    ),
+    name: 'firehose-quick-start',
+    categories: [],
+    icons: [
+      {
+        type: 'svg',
+        src: 'https://epr.elastic.co/package/awsfirehose/1.1.0/img/logo_firehose.svg',
```

Comment:
Can we use the `PackageIcon` component here as well?

Reply:
It's a bit cumbersome to use in our case. `PackageIcon` requires a full package definition from [the search endpoint](https://epr.elastic.co/search) which we don't fetch. I understand hardcoding those URLs does not feel super clean, but it's the best compromise I could think of between copying the icon into our plugin and adding more logic to fetch full package info just to show the icon.